### DISCUSSION TITLE
I want to use nearLine storage data for three months expired。How to achieve it，I have tried a number of ways，can help me ？

### DISCUSSION THREAD

**Discussion by lxb0425** [dcm4che/dcm4chee-arc-light#3962] [General]
I have followed to do，But it didn't work 
https://github.com/dcm4che/dcm4chee-arc-light/wiki/Nearline-Storage

Please describe the details 
thanks very much

**Comment by vrindanayak** [comment]
@lxb0425 : Your query 
> I want to use nearLine storage data for three months expired

is not quite clear. What exactly is the use case you are trying to achieve?
